react/finger-exercises/ecmascript/01 - spread/index.js

```diff
@@ -0,0 +1,24 @@
+import { isArray } from './utils';
+
+export function min(...args) {
+  if(args.length && isArray(...args)){
+    const [newArgs] = args;
+    return Math.min(...newArgs);
+  } else if (args.length) {
+    return Math.min(...args);
+  } else 
```

Comment:
This else is not required since when not matching the previous cases it will end up here anyway. 😺